app/src/main/java/org/groundplatform/android/ui/datacollection/tasks/AbstractTaskMapFragment.kt

```diff
@@ -145,6 +145,11 @@ abstract class AbstractTaskMapFragment<TVM : AbstractTaskViewModel> :
       }
     }
 
+  fun updateCenterMarker(isTooClose: Boolean, isMarkedComplete: Boolean) {
+    val shouldShow = !isMarkedComplete && !isTooClose
+    binding.centerMarker.visibility = if (shouldShow) View.VISIBLE else View.GONE
+  }
```

Comment:
Can we change this method to:

```
fun setCenterMarkerVisibility(visible: Boolean) {
   binding.centerMarker.visibility = if (visible) View.VISIBLE else View.GONE
}
```

This would help simplify the structure of the class.